babel-preset/transforms/transform-remove-console.js

```diff
@@ -0,0 +1,17 @@
+var template = require('babel-template');
+module.exports = function () {
+  return {
+    visitor: {
+      CallExpression(path) {
+        const callee = path.get("callee")
+        if (callee.matchesPattern("console", true) && !callee.matchesPattern("console.error")) {
+          path.node.callee = template(`
+           (function () {
```

Comment:
Instead of creating a new function object each time, it might be simpler just to use `Function.prototype`, which is a no-op function #jstrivia